netconf_client/connect.py

```diff
@@ -172,6 +177,15 @@ def __exit__(self, _, __, ___):
         self.server_socket.close()
 
 
+def _try_load_hostkey_b64(data):
+    for cls in (paramiko.RSAKey, paramiko.DSSKey, paramiko.ECDSAKey):
+        try:
+            return cls(data=b64decode(data))
+        except paramiko.SSHException:
+            pass
+    return None
```

Comment:
I'm wondering if this should throw some kind of exception instead of returning `None`. Otherwise, if the host key happens to fail to load it will silently do no verification even though a (presumably broken) key was passed to `connect_ssh`.

Reply:
Sounds good, I'll add a new exception in error.py and raise it here?

Reply:
Yep that works for me!

Reply:
Thanks, updated!